### PR TITLE
Update README and CONTRIBUTING files

### PR DESCRIPTION
**Pull request by racbc**
These updates should facilitate contributions by developers.

CANCELS #91 AS I REMOVED MOST OF THE TEXT IN THE README FILE, INCLUDING IMAGE REFERENCES. 
(I canceled that PR.)

### Motivation
These edits are meant to facilitate contributions by OSS developers. 

### Solution
Updated the README and CONTRIBUTING files to be more specific about how and where to contribute. Also added references to additional Rosetta repos.

### Open questions
I welcome any suggestions in the comments.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,37 +52,45 @@ To submit a feature request, follow these steps:
...
+#### Pull Requests
 
-* Ensure your [commit messages are well-written][2]. This can double as your pull request message, so it pays to take the time to write a clear message.
+Before creating your pull request, it's usually worth asking whether the code you're planning on writing will be considered for merging. You can do this by [opening an issue][1] and asking. It may also help give the maintainers context for when the time comes to review your code.
 
-* Add tests for your feature. You should be able to [look at other tests][4] for examples. If you're unsure, don't hesitate to [open an issue][1] and ask!
+Ensure that your [commit messages are well-written][2]. This can double as your pull request message, so it pays to take the time to write a clear message.
 
-* Submit your pull request!
+Additionally, make sure that you have written unit tests for your changes. If you're unsure as to what to test, don't hesitate to [open an issue][1] and ask!
```
Comment (shrimalmadhur) [inline CONTRIBUTING.md:61]: Actually we don't need unit tests for this repo and this is just a specification repo so we can remove it

File: README.md [hunk 2]
```diff
@@ -14,406 +14,61 @@ This repository contains all specification files used to generate code for the R
...
-access to private keys and will only ever interact with public keys and signatures
-in standardized formats. These formats are listed below:
+## Related Projects
 
-_CurveType and SignatureType are purposely decoupled as a curve could be used
-with multiple signature types (i.e. `secp256k1:ECDSA` and `secp256k1:Schnorr`)._
+* [rosetta-sdk-go][https://github.com/coinbase/rosetta-sdk-go] — The `rosetta-sdk-go` SDK provides a collection of packages used for interaction with the Rosetta API specification. Much of the SDK code is generated from this, the [rosetta-specifications](https://github.com/coinbase/rosetta-specifications) repository.
+* [rosetta-cli][https://github.com/coinbase/rosetta-ecosystem] — Use the `rosetta-cli` tool to test your Rosetta API implementation. The tool also provides the ability to look up block contents and account balances.
 
-#### Supported CurveTypes
```
Comment (shrimalmadhur) [inline README.md:309]: Where are all these docs moving to?

Comment (racbc) [reply README.md:309]: They would move to rosetta-api.org. Just not sure where in there to put them.

File: CONTRIBUTING.md (hunk 1 above)
Comment (racbc) [reply CONTRIBUTING.md:61]: Removed in next commit.

File: README.md [hunk 3]
```diff
@@ -14,406 +14,61 @@ This repository contains all specification files used to generate code for the R
...
-### CurveTypes and SignatureTypes
-The Construction API only supports detached, curve-based key generation and signing.
-In short, this means that Construction API implementations will never have
-access to private keys and will only ever interact with public keys and signatures
-in standardized formats. These formats are listed below:
+## Related Projects
 
-_CurveType and SignatureType are purposely decoupled as a curve could be used
-with multiple signature types (i.e. `secp256k1:ECDSA` and `secp256k1:Schnorr`)._
+* [rosetta-sdk-go][https://github.com/coinbase/rosetta-sdk-go] — The `rosetta-sdk-go` SDK provides a collection of packages used for interaction with the Rosetta API specification. Much of the SDK code is generated from this, the [rosetta-specifications](https://github.com/coinbase/rosetta-specifications) repository.
```
Comment (shrimalmadhur) [inline README.md:59]: The .md format for links is wrong. [rosetta-sdk-go](link)

Comment (racbc) [reply README.md:59]: Fixed in next commit.

**Review by shrimalmadhur — APPROVED** [review]

**Pull request merged**